modules/optimeraBidAdapter.js

```diff
@@ -0,0 +1,75 @@
+import {registerBidder} from 'src/adapters/bidderFactory';
+const BIDDER_CODE = 'optimera';
+const SCORES_BASE_URL = 'https://s3.amazonaws.com/elasticbeanstalk-us-east-1-397719490216/json/client/';
+
+export const spec = {
+  code: BIDDER_CODE,
+  /**
+   * Determines whether or not the given bid request is valid.
+   *
+   * @param {bidRequest} bid The bid params to validate.
+   * @return boolean True if this is a valid bid, and false otherwise.
+   */
+  isBidRequestValid: function (bidRequest) {
+    return !!(bidRequest.params.custom.clientID);
+  },
+  /**
+   * Make a server request from the list of BidRequests.
+   *
+   * @param {validBidRequests[]} - an array of bids
+   * @return ServerRequest Info describing the request to the server.
+   */
+  buildRequests: function (validBidRequests) {
+    var optimeraHost = window.location.host;
+    var optimeraPathName = window.location.pathname;
+    var timestamp = Math.round(new Date().getTime() / 1000);
+    var clientID = validBidRequests[0].params.custom.clientID;
+    var oDv = [];
+    if (clientID != undefined) {
+      oDv.push(clientID);
+      for (var i = 0; i < validBidRequests.length; i++) {
+        oDv.push(validBidRequests[i].placementCode);
+      }
+      window.oDv = oDv;
+      var scoresURL = SCORES_BASE_URL + clientID + '/' + optimeraHost + optimeraPathName + '.js';
```

Comment:
This looks like JSONP. 

1.0 adapters should use AJAX capable endpoints.

Reply:
For this call, the platform generates a static js file with the JSON set to a var, so there isn't a callback. We are able to parse it from ```serverRsponse.body``` in ```interpretResponse```.  We were hoping as long as this works ok it would be fine for now until the platform can be updated.